src/main/java/com/mg105/use_cases/ReplayGenerator.java

```diff
@@ -35,7 +36,8 @@ public void inventoryClean() {
      * Replay the game by interacting with the MapGenerator class
      */
     public void replay() {
-        inventoryClean();
+        this.inventoryClean();
+        this.attributeInheritance();
```

Comment:
If this (and inventoryClean) are only called as helper methods, shouldn't they be private?